include/neatvnc.h

```diff
@@ -111,3 +113,6 @@ void nvnc_display_damage_whole(struct nvnc_display*);
 int nvnc_check_damage(struct nvnc_fb* fb0, struct nvnc_fb* fb1,
                       int x_hint, int y_hint, int width_hint, int height_hint,
                       nvnc_damage_fn on_check_done, void* userdata);
+
+void nvnc_server_cut_text(struct nvnc_display*, char *text, uint32_t length);
+void _nvnc_server_cut_text(struct nvnc_display*, char *text, uint32_t length);
```

Comment:
text should be const and the star should be on the left side of the space. Also, does there need to be length? `_nvnc_server_cut_text` should not be here as it is not a public API function.

Reply:
Also, I'd prefer the name `nvnc_send_cut_text`. The names "client cut text" and "server cut text" are implementation details leaking from the protocol.